### PR TITLE
Change password page

### PR DESCRIPTION
**Pull request by Zebedeusz**

### CODE REVIEW

File: .vscode/snipsnap.code-snippets [hunk 1]
```diff
@@ -0,0 +1 @@
... console time end"],"body":["console.timeEnd(${1:object});"],"description":"Sets end point for execution time measurement","scope":"javascript,javascriptreact,typescript,typescriptreact"},"settimeout-1eb369f":{"prefix":["base set timeout"],"body":["setTimeout(() => {\n\t${0}\n}, ${1:delay});"],"description":"setTimeout","scope":"javascript,javascriptreact,typescript,typescriptreact"},"setinterval-25bbd90":{"prefix":["base set interval"],"body":["setInterval(() => {\n\t${0}\n}, ${1:delay});"],"description":"setInterval","scope":"javascript,javascriptreact,typescript,typescriptreact"},"setimmediate-44177fa":{"prefix":["base set immediate"],"body":["setImmediate(() => {\n\t${0}\n});"],"description":"setImmediate","scope":"javascript,javascriptreact,typescript,typescriptreact"},"process-nexttick-9043b4d":{"prefix":["base process next tick"],"body":["process.nextTick(() => {\n\t${0}\n});"],"description":"process nextTick","scope":"javascript,javascriptreact,typescript,typescriptreact"},"insert-use-strict-statement-dbe202c":{"prefix":["base use strict"],"body":["'use strict';"],"description":"insert 'use strict' statement","scope":"javascript,javascriptreact,typescript,typescriptreact"}}
```
Comment (kebetsi) [inline .vscode/snipsnap.code-snippets:1]: are you supposed to commit this? isn't this part of your personal workspace?

File: tests/e2e/changePassword.test.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+const authReqLogger = RequestLogger(authEndpoint, {
+  logRequestBody: true,
+  stringifyRequestBody: true,
+});
+const changePasswordReqLogger = RequestLogger(changePasswordEndpoint, {
+  logRequestBody: true,
+  stringifyRequestBody: true,
+});
+
+const authRequestMock = RequestMock()
```
Comment (kebetsi) [inline tests/e2e/changePassword.test.js:16]: autLoginMock

File: .vscode/snipsnap.code-snippets (hunk 1 above)
Comment (Zebedeusz) [reply .vscode/snipsnap.code-snippets]: Fixed

File: tests/e2e/changePassword.test.js (hunk 2 above)
Comment (Zebedeusz) [reply tests/e2e/changePassword.test.js]: Done

**Pull request merged**